ui/js/controllers/perf/compare.js

```diff
@@ -571,10 +569,10 @@ perf.controller('CompareSubtestResultsCtrl', [
                 if ($scope.originalRevision) {
                     cmap.links = [{
                         title: 'graph',
-                        href: PhCompare.getGraphsLink(_.map(_.uniq([
+                        href: PhCompare.getGraphsLink(_.uniq([
                             $scope.originalProject,
                             $scope.newProject
-                        ]), function (project) {
+                        ]).map(function (project) {
```

Comment:
Wherever you modify a line that has ``function`` would you change it to an arrow function instead?  We're  migrating to that usage all over.  Thanks.  :)